index.js

```diff
@@ -0,0 +1,5 @@
+var connect = require('connect');
+var serveStatic = require('serve-static');
+connect().use(serveStatic(__dirname)).listen(8080, function(){
```

Comment:
Do we need this?

Reply:
I guess we need this to make it work locally?

Reply:
During development there was a distinct difference between testing the html file loaded straight from the filesystem and hosted on a server, this was to make it more representative of it running live.